src/plugins/cpuidmon/cpuidmon.cpp

```diff
@@ -179,6 +179,8 @@ cpuidmon::cpuidmon(drakvuf_t _drakvuf, bool _stealth, output_format_t _output)
     this->cpuid.data = (void*)this;
     this->cpuid.type = CPUID;
     this->cpuid.name = nullptr;
+    this->cpuid.ttl = UNLIMITED_TTL;
+    this->cpuid.ah_cb = nullptr;
```

Comment:
Unneeded inits

Reply:
> Thanks for making the change! Please just remove the unneeded inits now for cases where the base struct is known to be already 0 initialized.

I will resolve all of the above suggestions at the beginning of next week;) Thanks for through cr.

Reply:
I think this has to stay as it is. Same reason as for debugmon.